src/pages/Onboarding/Onboarding.tsx

```diff
@@ -92,7 +92,15 @@ const Onboarding = (): JSX.Element => {
     },
     {
       title: 'Request Funds',
-      content: 'If you do not have any INTR, join our Discord and request funds in the #faucet channel.',
+      content: (
+        <>
+          If you do not have any INTR, join our Discord and request funds in the{' '}
+          <TextLink external weight='bold' color='secondary' to='#'>
```

Comment:
Would be useful highlighting the channel. This could be done with a link that use can use and also a single bold